presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/MetastoreLocator.java

```diff
@@ -15,19 +15,10 @@
 
 import org.apache.thrift.TException;
 
-/**
- * A Hive cluster is a single logical installation of Hive. It might
- * have multiple instances of the metastore service (for scalability
- * purposes), but they would all return the same data.
- * <p/>
- * This Hive plugin only supports having a single Hive cluster per
- * instantiation of the plugin, but a plugin that extends this code
- * could support multiple, dynamically located Hive clusters.
- */
-public interface HiveCluster
+public interface MetastoreLocator
```

Comment:
`MetastoreClientFactory` ?

Reply:
That was the first name I picked, but it's confusing since we already have `HiveMetastoreClientFactory`. We should probably refactor this further as I don't think we need two interfaces.